python/nav/ipdevpoll/plugins/bridge.py

```diff
@@ -34,9 +37,20 @@ class Bridge(Plugin):
     @defer.inlineCallbacks
     def handle(self):
         bridge = yield get_multibridgemib(self.agent)
+        bridge_address = yield bridge.get_base_bridge_address()
+        self._save_bridge_address(bridge_address)
         baseports = yield bridge.get_baseport_ifindex_map()
         defer.returnValue(self._set_port_numbers(baseports))
 
+    def _save_bridge_address(self, bridge_address):
```

Comment:
I think you need to verify there is an actual value to store here. I immediately get several of these when running on a copy of a prod database:

```ERROR:  null value in column "val" violates not-null constraint
DETAIL:  Failing row contains (2043, 93, bridge_info, base_address, null).
STATEMENT:  INSERT INTO "netboxinfo" ("netboxid", "key", "var", "val") VALUES (93, 'bridge_info', 'base_address', NULL) RETURNING "netboxinfo"."netboxinfoid"
```